lib/addons/p5.dom.js

```diff
@@ -550,12 +554,19 @@
    * }
    * </code></div>
    */
+
   p5.prototype.createSelect = function(mult) {
-    var elt = document.createElement('select');
-    if (mult){
-      elt.setAttribute('multiple', 'true');
+
+    if( typeof mult === 'object' && mult.elt.nodeName == 'SELECT' ){
+      var self = mult;
+      var elt = this.elt = mult.elt;
```

Comment:
If create select will be called via wrapElement, that means the argument mult should be a p5element. And if it is, than we just need the two methods of select box instead of creating a whole new select in Dom.